linchpin/provision/roles/libvirt/templates/cloud-config/user-data-local

```diff
@@ -6,6 +6,7 @@ ssh_authorized_keys:
   - "{{ pubkey_local['stdout'] }}"
 
 # Add users to the system. Users are added after groups are added.
+{% if 'cloud_config' in res_def %}
```

Comment:
why process this template at all if cloud_config is not defined?

Reply:
@p3ck : according to all the other tasks running these templates are processed only when the virt-type is defined. 
So I think the if condition here is not necessary